lib/runner/extensions/http-request.command.js

```diff
@@ -226,7 +226,6 @@ module.exports = {
 
                         // run the post request helpers, which need to use the response, assigned above
                         async.applyEachSeries(posthelpers, context, self, function (error, options) {
-                            // bucket options per helper
```

Comment:
Why was the comment removed? :/

Reply:
It wasn't relevant to the code. It shouldn't have been there in the first place.